### ISSUE TITLE
Contrasto insufficiente nella descrizione delle card

### ISSUE THREAD

**Issue by martalucivero** [italia/graph-italia#130]
### Esiste già una discussione sul tema che ti interessa, o su un tema simile?

- [x] Ho verificato e non esiste

### Cosa

Il testo in preview e le etichette delle card accordion per la gestione di grafici, dashboard, ecc. non hanno contrasto sufficiente.

### Perché

L'elemento ha un contrasto di colore insufficiente di 4.18 (colore in primo piano: #7a7c7f, colore di sfondo: #ffffff, dimensione del font: 12.0pt (16px), peso del font: normal). Contrasto atteso di 4.5:1

WCAG 1.4.3 – Contrast (Minimum) (Livello AA): il testo deve avere un rapporto di contrasto sufficiente rispetto allo sfondo.

### Contesto

<img width="402" height="260" alt="Image" src="https://github.com/user-attachments/assets/bb54e8a0-32d7-4924-b348-b908f88f1e8e" />

<img width="404" height="828" alt="Image" src="https://github.com/user-attachments/assets/50d1b8f1-cd9c-4935-847a-afadf1bec7fe" />

**Comment by martalucivero** [comment]
@AlexPalaz risolto parzialmente, le etichette come ad es. "Direzione" non hanno ancora un contrasto sufficiente
